assets/js/components/DashboardMainApp.js

```diff
@@ -313,9 +313,7 @@ export default function DashboardMainApp() {
 
 			{ showSurveyPortal && <CurrentSurveyPortal /> }
 
-			{ showKeyMetricsSelectionPanel === false && (
-				<MetricsSelectionPanel />
-			) }
+			{ showKeyMetricsSelectionPanel && <MetricsSelectionPanel /> }
```

Comment:
This will be removed when https://github.com/google/site-kit-wp/pull/9630 will be merged.